sql/setup.sql

```diff
@@ -0,0 +1,11 @@
+ DROP TABLE if EXISTS booksTable;
+
+ CREATE TABLE booksTable (
+  id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  img TEXT NOT NULL,
+  rating TEXT, 
+  title TEXT,
+  price INTEGER NOT NULL,
+  stock BOOLEAN NOT NULL,
+
+);
```

Comment:
When you put this in PGadmin, it fails. That should be a good hint that this isn't valid SQL. Hint: check your comma before the ).